tests/e2e/README.MD

```diff
@@ -0,0 +1,37 @@
+For VS Code to work correctly with e2e, following settings must be added because of build constraints:
+
+```json
+    "gopls": {
+         "build.buildFlags": [
+             "-tags=e2e"
+         ]
+     },
+    "go.useLanguageServer": true,
+```
+
+Example launch json debug config:
+
+```json
+{ // e2e
```

Comment:
Does this work including the comments? 
JSON doesn't define/allow comments

Reply:
review came a bit late :P its already `jsonc`